File: testdata/python3/book/create14.py
#!/usr/bin/env python
# -*- coding:utf-8 -*-
"""
function to say hi in python
"""

import json

def handler(event, context):
    raise Exception("test custom error")
    resp = {
        "isBase64Encoded": False,
        "statusCode": 200,
        "headers": {
	        "Content-Type": "application/json"
        },
        "body": 's'
    }
    return resp

File: testdata/node10/book/create14.js
#!/usr/bin/env node

exports.handler = (event, context, callback) => {
    throw new Error('test custom error')
    let obj = {
        "isBase64Encoded": false,
        "statusCode": 200,
        "headers": {
            "Content-Type": "application/json"
        },
        "body": 's'
    }
    callback(null, JSON.stringify(obj));
};